google/cloud/documentai_toolbox/wrappers/page.py

```diff
@@ -400,7 +566,13 @@ class Page:
 
     def __post_init__(self):
         tables = []
-
+        """
+        Order of Init
+        Token
+        Line
+        Paragraph,
+        Block
+        """
         for table in self.documentai_page.tables:
```

Comment:
If we are going to rename `documentai_line` to `documentai_object` as you have done in other places, let's keep this consistent and change it for the `Page` class too.